source/features/reactions-avatars.tsx

```diff
@@ -80,7 +104,10 @@ function observeCommentReactions(commentReactions: Element): void {
 }
 
 function init(signal: AbortSignal): void {
-	observe('.has-reactions .js-comment-reactions-options', observeCommentReactions, {signal});
+	observe([
+		'.has-reactions .js-comment-reactions-options', // TODO: Remove after March 2024
```

Comment:
It seems that the feature supports discussions too, so shouldn't drop these. 

The second line can be marked with `// React views, isIssue`